packages/trader/src/Modules/Trading/Containers/trade.jsx

```diff
@@ -5,8 +5,9 @@ import ChartLoader from 'App/Components/Elements/chart-loader.jsx';
 import { connect } from 'Stores/connect';
 import PositionsDrawer from 'App/Components/Elements/PositionsDrawer';
 import MarketIsClosedOverlay from 'App/Components/Elements/market-is-closed-overlay.jsx';
-import Test from './test.jsx';
+// import Test from './test.jsx';
```

Comment:
Can u uncomment this? We might still need it for debugging on Desktop. Maybe use `DesktopWrapper` instead for `<Test />`.